### PR TITLE
BEing fixes

### PR DESCRIPTION
**Pull request by stanalbatross**
## About The Pull Request

fixes an errant return

## Changelog

:cl:
fix: BEs will now actually fire bullets again
/:cl:

### CODE REVIEW

File: code/modules/projectiles/ammo_datums.dm [hunk 1]
```diff
@@ -287,7 +287,6 @@
 
 		if(flags_ammo_behavior & AMMO_EXPLOSIVE)
 			execution_target.gib()
-		return TRUE
 	return ..()
```
Comment (spookydonut) [inline code/modules/projectiles/ammo_datums.dm:290]: this whole proc can be early returned at the top with `if(!(flags_ammo_behavior & AMMO_HIGHIMPACT)) return ..()`

then keep the `return ..()` at the bottom and the whole thing is no longer indented and you remove the issue that created this code oversight bug

File: code/modules/projectiles/ammo_datums.dm [hunk 2]
```diff
@@ -241,53 +241,54 @@
...
-		log_attack("[key_name(usr)] battlefield executed [key_name(execution_target)] at [execution_area.name].")
+	msg_admin_attack(FONT_SIZE_HUGE("[key_name(usr)] has battlefield executed [key_name(execution_target)] in [get_area(usr)] ([usr.loc.x],[usr.loc.y],[usr.loc.z])."), usr.loc.x, usr.loc.y, usr.loc.z)
+	log_attack("[key_name(usr)] battlefield executed [key_name(execution_target)] at [execution_area.name].")
 
-		if(flags_ammo_behavior & AMMO_EXPLOSIVE)
-			execution_target.gib()
-		return TRUE
+	if(flags_ammo_behavior & AMMO_EXPLOSIVE)
+		execution_target.gib()
 	return ..()
```
Comment (Geevies) [inline code/modules/projectiles/ammo_datums.dm:292]: does this final return ..() make the gun fire normally after execution?

Comment (stanalbatross) [reply code/modules/projectiles/ammo_datums.dm:292]: yes

File: code/modules/projectiles/ammo_datums.dm [hunk 3]
```diff
@@ -241,53 +241,54 @@
...
+	if(!skillcheck(user, SKILL_EXECUTION, SKILL_EXECUTION_TRAINED))
+		to_chat(user, SPAN_DANGER("You don't know how to execute someone correctly."))
+		return FALSE
 
-		user.affected_message(execution_target,
-			SPAN_HIGHDANGER("You aim \the [fired_from] at [execution_target]'s head!"),
-			SPAN_HIGHDANGER("[user] aims \the [fired_from] directly at your head!"),
-			SPAN_DANGER("[user] aims \the [fired_from] at [execution_target]'s head!"))
+	if(execution_target.status_flags & PERMANENTLY_DEAD)
+		to_chat(user, SPAN_DANGER("[execution_target] is already as dead as it's possible to be!"))
```
Comment (Shad0vvs) [inline code/modules/projectiles/ammo_datums.dm:259]:
```suggestion
		to_chat(user, SPAN_DANGER("[execution_target] is already as dead!"))
```

**Review by Shad0vvs — APPROVED** [review]

**Pull request merged**